packages/commonwealth/client/scripts/state/api/profiles/fetchProfileById.ts

```diff
@@ -1,113 +1,25 @@
-import { GetNewProfileResp } from '@hicommonwealth/schemas';
-import { useQuery } from '@tanstack/react-query';
-import axios from 'axios';
-import AddressInfo from 'models/AddressInfo';
-import moment from 'moment';
-import { ApiEndpoints, SERVER_URL } from 'state/api/config';
-import { z } from 'zod';
-import useUserStore, { userStore } from '../../ui/user';
+import { trpc } from 'client/scripts/utils/trpcClient';
 
 const PROFILE_STALE_TIME = 30 * 1_000; // 3 minutes
 
-type UseFetchProfileByIdQueryCommonProps =
-  | {
-      userId: number;
-      shouldFetchSelfProfile?: never;
-    }
-  | {
-      userId?: never;
-      shouldFetchSelfProfile: boolean;
-    };
-
-const fetchProfileById = async ({
-  userId,
-}: UseFetchProfileByIdQueryCommonProps): Promise<
-  z.infer<typeof GetNewProfileResp>
-> => {
-  const response = await axios.get<{
-    result: z.infer<typeof GetNewProfileResp>;
-  }>(`${SERVER_URL}${ApiEndpoints.FETCH_PROFILES_BY_ID}`, {
-    params: {
-      ...(userId
-        ? { userId }
-        : {
-            jwt: userStore.getState().jwt,
-          }),
-    },
-  });
-
-  response.data.result.addresses.map((a) => {
-    try {
-      return new AddressInfo({
-        userId: userStore.getState().id,
-        id: a.id!,
-        address: a.address,
-        community: {
-          id: a.community_id!,
-          base: a.Community.base,
-          ss58Prefix: a.Community.ss58_prefix || undefined,
-        },
-        walletId: a.wallet_id!,
-        ghostAddress: a.ghost_address,
-      });
-    } catch (err) {
-      console.error(`Could not return AddressInfo: "${err}"`);
-      return null;
-    }
-  });
-
-  return response.data.result;
+type UseFetchProfileByIdQueryCommonProps = {
+  userId?: number;
+  apiCallEnabled?: boolean;
 };
 
-interface UseFetchProfileByIdQuery {
-  apiCallEnabled?: boolean;
-}
 const useFetchProfileByIdQuery = ({
   userId,
-  shouldFetchSelfProfile,
   apiCallEnabled = true,
-}: UseFetchProfileByIdQuery & UseFetchProfileByIdQueryCommonProps) => {
-  const user = useUserStore();
-
-  return useQuery({
-    // eslint-disable-next-line @tanstack/query/exhaustive-deps
-    queryKey: [ApiEndpoints.FETCH_PROFILES_BY_ID, userId],
-    queryFn: () =>
-      fetchProfileById({
-        userId,
-        shouldFetchSelfProfile,
-      } as UseFetchProfileByIdQueryCommonProps),
-    // eslint-disable-next-line @tanstack/query/no-deprecated-options
-    onSuccess: (response) => {
```

Comment:
removing this causes a bug with edit profile page

https://github.com/user-attachments/assets/f60977e6-c93d-4350-88d2-5593de184d0e

Reply:
@mzparacha did you fix this? or just reporting the bug here? ... do you know how to fix it quickly if still a bug?

Reply:
for more context, I removed it b/c it was creating an infinite loop in the client

Reply:
> did you fix this?

nope, but going to give it a try locally.

Reply:
added https://github.com/hicommonwealth/commonwealth/pull/9705/commits/59ad64b4bb4d254bc1985c722524e28dfcd85e96, which should fix this